src/gis/components/Map.js

```diff
@@ -1,9 +1,79 @@
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import { GeoJSON, MapContainer, TileLayer, useMap } from 'react-leaflet';
 import { v4 as uuidv4 } from 'uuid';
+import { GeoSearchControl, OpenStreetMapProvider } from 'leaflet-geosearch';
+import L from 'leaflet';
+
+import 'leaflet/dist/leaflet.css';
+import 'leaflet-geosearch/dist/geosearch.css';
 
 import 'gis/components/Map.css';
 
+delete L.Icon.Default.prototype._getIconUrl;
```

Comment:
Question: Why is this needed?

Reply:
If we don't include it, the marker icon is broken.